internal/gcs/gcs_upload.go

```diff
@@ -44,22 +45,24 @@ func (u *GCSUpload) UploadFile(filename string, contents []byte) error {
 		return errors.New("please initialise the connection")
 	}
 	bucket := viper.GetString("BUCKET_NAME")
-	log.WithField("filename", filename).WithField("bucket", bucket).Info("uploading to bucket")
+	path := bucketPath(filename)
+
+	log.WithField("filename", path).WithField("bucket", bucket).Info("uploading to bucket")
 
 	ctx, cancel := context.WithTimeout(u.ctx, time.Second*50)
 	defer cancel()
 
 	// GCSUpload an object with storage.Writer.
-	wc := u.client.Bucket(bucket).Object(filename).NewWriter(ctx)
-	log.WithField("filename", filename).WithField("bucket", bucket).Info("about to write contents to bucket")
+	wc := u.client.Bucket(bucket).Object(path).NewWriter(ctx)
+	log.WithField("filename", path).WithField("bucket", bucket).Info("about to write contents to bucket")
 	if _, err := wc.Write(contents); err != nil {
-		log.WithError(err).Error("error writing bytes to bucket")
+		log.WithError(err).Error("error writing bytes to bucket " + bucket + path)
```

Comment:
this should be:

log.WithError(err).WithField("bucket", bucket).WithField("path", path).Error("error writing bytes to bucket ")

Reply:
Fixed.